### PR TITLE
Added op-scim-bridge

### PR DESCRIPTION
**Pull request by cohix**
This adds the solution for the 1Password SCIM bridge.

### CODE REVIEW

**Comment by paulanunda** [comment]
Thanks for the submission @cohix; we'll get it reviewed shortly.

File: stacks/op-scim-bridge/deploy-local.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# create namespace
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: op-scim-bridge
+EOF
+
+# set kubectl namespace
+# kubectl config set-context --current --namespace=op-scim-bridge
```
Comment (paulanunda) [inline stacks/op-scim-bridge/deploy-local.sh]: Commenting this out results in op-scim-bridge being installed into whatever namespace the installer happens to be in (`default` in my case). Mind uncommenting it?

File: stacks/op-scim-bridge/yaml/op-scim-bridge.yaml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  labels:
+    app.kubernetes.io/application: op-scim-bridge
+    app.kubernetes.io/component: op-scim-bridge
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Mi
+  storageClassName: standard
```
Comment (paulanunda) [inline stacks/op-scim-bridge/yaml/op-scim-bridge.yaml]: `standard` should be `do-block-storage`

File: stacks/op-scim-bridge/yaml/op-scim-bridge.yaml [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  name: op-scim-bridge-pvc
+  labels:
+    app.kubernetes.io/application: op-scim-bridge
+    app.kubernetes.io/component: op-scim-bridge
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Mi
```
Comment (paulanunda) [inline stacks/op-scim-bridge/yaml/op-scim-bridge.yaml]: The minimum size supported by DigitalOcean is `1Gi`

File: stacks/op-scim-bridge/deploy-local.sh (hunk 1 above)
Comment (cohix) [reply stacks/op-scim-bridge/deploy-local.sh]: Ah, the command generated an error for me, and I thought it only was used for my testing. Will uncomment.

File: stacks/op-scim-bridge/yaml/op-scim-bridge.yaml (hunk 2 above)
Comment (cohix) [reply stacks/op-scim-bridge/yaml/op-scim-bridge.yaml]: done

File: stacks/op-scim-bridge/yaml/op-scim-bridge.yaml (hunk 3 above)
Comment (cohix) [reply stacks/op-scim-bridge/yaml/op-scim-bridge.yaml]: done

**Pull request merged**